examples/drive/flux_vector/plot_flux_vector_pmsm_2kw.py

```diff
@@ -21,28 +26,27 @@
 # %%
 # Configure the system model.
 
-mdl_par = SynchronousMachinePars(
-    n_p=3, R_s=3.6, L_d=.036, L_q=.051, psi_f=.545)
+mdl_par = SynchronousMachinePars(n_p=3, R_s=3.6, L_d=0.036, L_q=0.051, psi_f=0.545)
 machine = model.SynchronousMachine(mdl_par)
-mechanics = model.StiffMechanicalSystem(J=.015)
+mechanics = model.StiffMechanicalSystem(J=0.015)
 converter = model.VoltageSourceConverter(u_dc=540)
 mdl = model.Drive(converter, machine, mechanics)
 
 # %%
 # Configure the control system.
 
 par = mdl_par  # Assume accurate machine model parameter estimates
-cfg = control.FluxTorqueReferenceCfg(par, max_i_s=1.5*base.i, k_u=.9)
-ctrl = control.FluxVectorControl(par, cfg, J=.015, T_s=250e-6, sensorless=True)
+cfg = control.FluxTorqueReferenceCfg(par, max_i_s=1.5 * base.i, k_u=0.9)
+ctrl = control.FluxVectorControl(par, cfg, J=0.015, T_s=250e-6, sensorless=True)
 
 # %%
 # Set the speed reference and the external load torque.
 
 # Speed reference (electrical rad/s)
-ctrl.ref.w_m = lambda t: (t > .2)*2*base.w
+ctrl.ref.w_m = lambda t: (t > 0.2) * 2 * base.w
 
 # Load torque step
-mdl.mechanics.tau_L = lambda t: (t > .8)*nom.tau*.7
+mdl.mechanics.tau_L = lambda t: (t > 0.8) * nom.tau * 0.7
```

Comment:
It seems that these modified files do not follow the yapf style defined in the pyproject.toml file for some reason.

Reply:
Is this related to the `no_spaces_around_selected_binary_operators = "*,/"` rule? If so, could you check the `diff` of commit https://github.com/Aalto-Electric-Drives/motulator/pull/149/commits/b993b67010212de4643981729b6ea68fe91ea66b?

I've now included `additional_dependencies: [ toml ]` in the YAPF pre-commit hook.